nim/chain.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "nim",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://ai.nim.network",
+    "pretty_name": "Nim Network",
+    "chain_id": "nim_1122-1",
+    "bech32_prefix": "nim",
+    "slip44": 60,
+    "node_home": "",
+    "daemon_name": "",
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "anim",
+                "low_gas_price": 20000000000,
+                "average_gas_price": 20000000000,
+                "high_gas_price": 20000000000
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "anim",
+                "exponent": 0,
```

Comment:
exponent and aliases don't go under staking tokens--that's what the detected error is